### PR TITLE
fixed swagger docs to updated routers

### PR DESCRIPTION
**Pull request by francishtknguyen**
# Description
Fixed swagger docs to reflect changes to routes in the new table schema. Also removed old swagger docs from older routes. Also changed name of router for swagger documentation. 

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: api/allIncidents/allincidentsRouter.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
+const express = require('express');
+const router = express.Router();
+const Incidents = require('./incidentsModel');
+const {
+  checkIncidentExists,
+  validateAndSanitizeIncidentQueries,
+} = require('../middleware');
+
+// TODO document shape of objects coming and going
```
Comment (JustinPeczenij) [inline api/allIncidents/allincidentsRouter.js:9]: Looks like you did this and did it well!

File: api/incidents/incidentsRouter.js [hunk 2]
```diff
@@ -365,60 +95,6 @@ router.get('/incident/:id', async (req, res) => {
...
- * Possible Query Strings:
- * /download?state=*StateName Here*
- * /download?startDate=*StartDate Here*&endDate=*End Date Here*
- * /download?state=*StateName Here*&start=*StartDate Here*&end=*End Date Here*
- * State name ex: "New York" default: null
- * Start Date ex: "05-13-2020" default: One year ago from Today
- * End Date ex: "12-04-2020" default: Today
- */
-
 router.get('/download', async (req, res) => {
```
Comment (JustinPeczenij) [inline api/incidents/incidentsRouter.js:98]: Looks like we'll have to get a download route in our new route.

**Review by JustinPeczenij — DISMISSED** [review]
Documentation looks really good, we'll need to add the /download endpoint to our new router to keep the functionality consolidated.

File: api/allIncidents/adminIncidentsRouter.js [hunk 3]
```diff
@@ -101,11 +101,11 @@ router.get('/incidents/:incident_id', checkIncidentExists, async (req, res) => {
 
 /**
  * @swagger
- * /{incident_id}:
- *  PUT:
+ * /adminincidents/incidents/{incident_id}:
+ *  put:
```
Comment (abe-one) [inline api/allIncidents/adminIncidentsRouter.js:105]: @francishtknguyen Do you have time to go over how this endpoint works? On Monday the 26th perhaps?

File: api/incidents/incidentsRouter.js (hunk 2 above)
Comment (abe-one) [reply api/incidents/incidentsRouter.js:98]: I was under the impression that the download function in the frontend simply convert the array of objects stored in local memory into a csv file?

**Review by abe-one — DISMISSED** [review]

File: api/allIncidents/allincidentsRouter.js (hunk 1 above)
Comment (francishtknguyen) [reply api/allIncidents/allincidentsRouter.js:9]: Yeah it was helpful having the older one to reference!

File: api/incidents/incidentsRouter.js (hunk 2 above)
Comment (francishtknguyen) [reply api/incidents/incidentsRouter.js:98]: I'll  have to look into this as well

File: api/allIncidents/adminIncidentsRouter.js (hunk 3 above)
Comment (francishtknguyen) [reply api/allIncidents/adminIncidentsRouter.js:105]: Yes that works for me,

File: api/allIncidents/allincidentsRouter.js [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+ *              properties:
+ *                data:
+ *                  type: array
+ *                  example: [
+  {
+    incident_id: 1,
+    incident_date: '2021-03-01T00:00:00.000Z',
+    tweet_id: '1366291653267513344',
+    user_name: 'shafiur',
+    desc:
```
Comment (abe-one) [inline api/allIncidents/allincidentsRouter.js]: desc -> description

Comment (francishtknguyen) [reply api/allIncidents/allincidentsRouter.js]: fixed and corrected, weird because I copied it directly from seed file

Comment (francishtknguyen) [reply api/allIncidents/allincidentsRouter.js]: aaaaanddd fixed in seeds as well

**Review by abe-one — APPROVED** [review]

**Review by abe-one — APPROVED** [review]

**Review by JustinPeczenij — APPROVED** [review]

**Pull request merged**